src/developer/web-api/tracker.md

```diff
@@ -1432,8 +1437,11 @@ Tracker export endpoints allow you to retrieve the previously imported objects w
 
 > **NOTE**
 >
-> - All these endpoints currently support `JSON`. `CSV` is only supported by tracked entities and
->   events.
+> - All these endpoints default to a `JSON` response content. `CSV` is only supported by tracked entities and events. 
+> - You can export a CSV file by adding the `Accept` header ***text/csv*** or ***application/csv*** to the request. 
+> - Compression support is also available in zip and gzip formats for some endpoints. 
```

Comment:
The ones just mentioned

Reply:
In

> Compression support is also available in zip and gzip formats for some endpoints. 

`some` is raising the question which ones. I cannot see from this diff if there would be some indentation in the bullets so it is clear which ones. Otherwise say which ones instead of `some` please.

Reply:
ah ok i thought you mentioned `All these endpoints`